client/src/qa/components/QuestionAndAnswerEntry.jsx

```diff
@@ -1,26 +1,59 @@
-import React, { useState } from 'react';
-import styled from 'styled-components';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+// import styled from 'styled-components';
 import QuestionEntry from './QuestionEntry';
 import AnswerEntry from './AnswerEntry';
 
 const QuestionAndAnswerEntry = ({ question, answers }) => {
-  const [slice, setSlice] = useState(2);
-  const initialAnswers = answers.slice(0, slice);
+  const [slicer, setSlice] = useState(2);
+  const initialAnswers = answers.slice(0, slicer);
   const [currentAnswers, setCurrentAnswers] = useState(initialAnswers);
   const numOfAnswersLeft = answers.length - currentAnswers.length;
+  const showMoreAnswers = () => numOfAnswersLeft > 0;
+
+  useEffect(() => {
+    setCurrentAnswers(answers.slice(0, slicer));
+  }, [slicer]);
 
   return (
     <div>
       <QuestionEntry question={question} />
-      {currentAnswers.map((answer, index) => (
-        <AnswerEntry key={index} answer={answer} />
+      {currentAnswers.map((answer) => (
+        <AnswerEntry key={answer.answer_id} answer={answer} />
       ))}
-      {numOfAnswersLeft
-        && <button onClick={() => {
-          // setCurrentAnswers(prevAnswers => ...prevAnswers, answers)
-        }} type="button">See More Answers</button>}
+      {showMoreAnswers()
+        && (
+          <button
+            onClick={() => setSlice((prevSlicer) => prevSlicer + 2)}
```

Comment:
I'm not too experienced with React Hooks. So i'm not sure if what you wrote works. After reviewing the react hooks page, I think you can simplify your line 27 to the code below.
onClick ={() => setSlice(slicer + 2) 
https://reactjs.org/docs/hooks-state.html -- updating state section

Reply:
Good point - made that change